app-emulation/sen/sen-0.5.1.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{4,5} )
+
+inherit distutils-r1
+
+DESCRIPTION="Terminal User Interface for docker engine"
+HOMEPAGE="https://github.com/TomasTomecek/sen"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+DEPEND="dev-python/urwid
```

Comment:
Dependencies on Python modules always require `[${PYTHON_USEDEP}]` to enforce implementation match.